dltpy/dltpy_receive.py

```diff
@@ -118,14 +120,27 @@ def main():
 
     host = args.host
     port = 3490
-
+    
     flt = cli_common.parse_filters(args.filter)
     out_fn = args.output
-    if ':' in host:
-        host, port = host.split(':')
+    
+    IP6_with_port = ']:' in host
+    IP4_with_port = (not IP6_with_port) and ('.' in host) and (':' in host)
+
+    if (IP6_with_port or IP4_with_port):
+        ip, _, port = host.rpartition(':')
+        host = ip.strip("[]") 
         port = int(port)
 
-    ar = AsyncReceiver((host, port), out_fn, flt)
+    try:
+      ip = socket.getaddrinfo(host,port)
+      af = ip[0][0] 
+
+    except:
+        print("Error: Can't resolve given host URL")
+        sys.exit()
```

Comment:
please use `raise` instead, so the original exception will still be visible to simplify toubleshooting
```suggestion
        raise
```